src/resources/your-label/section-one/styles.scss

```diff
@@ -0,0 +1,66 @@
+.lb {
```

Comment:
@ganapavz  @asifamingov 
is the styles here suitable, should we look at including some of these into the swe?